src/QPMotionConstr.cpp

```diff
@@ -113,13 +113,16 @@ MotionConstrCommon::ContactData::ContactData(const rbd::MultiBody & mb,
 
 MotionConstrCommon::MotionConstrCommon(const std::vector<rbd::MultiBody> & mbs, int robotIndex)
 : robotIndex_(robotIndex), alphaDBegin_(-1), nrDof_(mbs[robotIndex_].nrDof()), lambdaBegin_(-1), fd_(mbs[robotIndex_]),
-  fullJacLambda_(), jacTrans_(6, nrDof_), jacLambda_(), cont_(), curTorque_(nrDof_), A_(), AL_(nrDof_), AU_(nrDof_)
+  fullJacLambda_(), jacTrans_(6, nrDof_), jacLambda_(), cont_(), curTorque_(nrDof_), lastTorque_(nrDof_), A_(),
+  AL_(nrDof_), AU_(nrDof_)
 {
   assert(std::size_t(robotIndex_) < mbs.size() && robotIndex_ >= 0);
+  lastTorque_.setZero(); // TODO should be initialized with a gravity-compensation torque?
```

Comment:
I think in general it is not possible to initialize this on the first run (because we'd need some contact force estimation) but it's also very unlikely that the first iteration would hit the torque derivative limits.